test/jdk/javax/swing/JRadioButton/8033699/bug8033699.java

```diff
@@ -33,13 +33,12 @@
 import java.awt.Robot;
 import java.awt.event.ActionListener;
 import java.awt.event.KeyEvent;
-
 import javax.swing.BorderFactory;
```

Comment:
I'd rather preserve the blank line between `java.*` and `javax.*` packages. However, the list of imports isn't too long in either block.